README.md

```diff
@@ -42,26 +42,26 @@ Please see
 
```

Comment:
This explains what backends do, and a little bit about how they are implemented, but not why I should care about them, or the interface they provide to me. "backend" is a really overloaded term, so something that makes more specific would help... perhaps write a short example of a backend?